src/shared/components/case-editor/case-edit-page/case-edit-page.component.spec.ts

```diff
@@ -460,12 +467,17 @@ describe('CaseEditPageComponent', () => {
         'cancelled': cancelled,
         'validate': (caseEventData: CaseEventData) => of(caseEventData),
         'saveDraft': (caseEventData: CaseEventData) => of(someObservable),
-        'caseDetails': {'case_id': '1234567812345678'},
+        'caseDetails': {'case_id': '1234567812345678', 'tabs': [], 'metadataFields': [], 'state': {'id': 'incompleteApplication', 'name': 'Incomplete Application', 'title_display': '# 1234567812345678: test'}},
```

Comment:
I know this is a test but can we format the object instead of sticking into one long line.

Reply:
formatted the object